### PR TITLE
apply markdown for output

### PR DESCRIPTION
**Pull request by prabirshrestha**
apply markdown syntax using `vim-vital-vs` https://github.com/prabirshrestha/vim-lsp/issues/865

### CODE REVIEW

**Comment by prabirshrestha** [comment]
@hrsh7th can you have a look at this if I'm using markdown correctly. It is working for me. 

Temporary fix until we have https://github.com/prabirshrestha/vim-lsp/pull/1078

File: autoload/lsp/ui/vim/output.vim [hunk 1]
```diff
@@ -2,6 +2,11 @@ let s:use_vim_popup = has('patch-8.1.1517') && g:lsp_preview_float && !has('nvim
 let s:use_nvim_float = exists('*nvim_open_win') && g:lsp_preview_float && has('nvim')
 let s:use_preview = !s:use_vim_popup && !s:use_nvim_float
 
+let s:Markdown = vital#lsp#import('VS.Vim.Syntax.Markdown')
+let s:MarkupContent = vital#lsp#import('VS.LSP.MarkupContent')
+let s:Window = vital#lsp#import('VS.Vim.Window')
+let s:Text = vital#lsp#import('VS.LSP.Text')
+
```
Comment (hrsh7th) [inline autoload/lsp/ui/vim/output.vim:9]: I'm worrying if it hit the performance or not.

I will investigate it.

Comment (prabirshrestha) [reply autoload/lsp/ui/vim/output.vim:9]: do u mean performance of import? If that is the case we could check if `s:Markdown` exists and then only load it else don't load it at all.

or do u mean perf or `Markdown.apply`? or `MarkupContent.normalize`?

Comment (hrsh7th) [reply autoload/lsp/ui/vim/output.vim:9]: Yes. I mean the performance of import.

The Markdown.{apply,normalize} is not lightweight but we can't avoid it, I think.

Comment (prabirshrestha) [reply autoload/lsp/ui/vim/output.vim:12]: sounds good. lazy loaded markdown in the meantime.

**Pull request merged**